ext/hash/hash.c

```diff
@@ -568,7 +568,7 @@ static void php_hash_do_hash_hmac(
 
 		php_hash_bin2hex(ZSTR_VAL(hex_digest), (unsigned char *) ZSTR_VAL(digest), ops->digest_size);
 		ZSTR_VAL(hex_digest)[2 * ops->digest_size] = 0;
-		zend_string_release_ex(digest, 0);
+		zend_string_efree(digest);
```

Comment:
Does it make sense to also use this in the error path above where you have `zend_string_release(digest);` ?

Reply:
Oh, I missed that, indeed I'll change it.